internal/symbolz/symbolz.go

```diff
@@ -65,7 +65,7 @@ func Symbolize(p *profile.Profile, force bool, sources plugin.MappingSources, sy
 
 // Check whether path ends with one of the suffixes listed in
 // pprof_remote_servers.html from the gperftools distribution
-func hasLegacySuffix(path string) bool {
+func hasGperftoolsSuffix(path string) bool {
 	var suffixes = [...]string{
```

Comment:
Using `var suffixes = [...]string{` is not quite idiomatic. `suffixes := []string{` is more so.